kubernetes/assets/dashboards/kubernetes_daemonsets.json

```diff
@@ -0,0 +1,983 @@
+{
```

Comment:
Content looks good 👍 , although now we are trying to make dashboards more "blog-like" to provide more information on the side. Here is a link to the best practices: https://docs.google.com/presentation/d/1Qe4oFsORv50ynCicJ4O-C-jjw8iRyVziKZ3PTLctBtk/edit#slide=id.g8b0864f09e_0_28.